CONTRIBUTIONS.md

```diff
@@ -0,0 +1,49 @@
+# Contributing to pixi
+We would love for you to contribute to pixi and make it better with every PR merged!
+
+To make contributing as easy as possible we made pixi a pixi project :wink:.
+
+## Getting started
+Clone our project locally on your machine.
+
+```
+git clone https://github.com/prefix-dev/pixi.git
+```
+
+Because pixi is a rust project we only need to use `cargo` but keeping it as simple as possible we also made some pixi tasks to help you get started.
```

Comment:
```suggestion
Because pixi is a [Rust](https://www.rust-lang.org/) project we only need to use `cargo` but keeping it as simple as possible we also made some pixi tasks to help you get started.
```